ui/prometheus-plugin/src/plugins/variable.tsx

```diff
@@ -40,13 +40,26 @@ function StaticListVariableOptionEditor(props: OptionsEditorProps<StaticListVari
   return (
     <div>
       <Autocomplete
+        onPaste={(e) => {
+          // Append new values on paste
+          const v = e.clipboardData.getData('text/plain');
+          if (v) {
+            const values = v.split(',');
+            onChange(null, value.concat(values));
```

Comment:
Maybe a "future" thing, but should we be trying to de-dupe values or anything in this case? I guess there probably isn't anything stopping someone from entering duplicates manually now, so maybe a future validation topic.